code/components/citizen-server-impl/include/state/ServerGameState.h

```diff
@@ -801,6 +802,8 @@ struct GameStateClientData : public sync::ClientSyncDataBase
 
 	uint32_t routingBucket = 0;
 
+	float playerCullingRadius = 0.0f;
```

Comment:
This will not be used as you're only reading the value from the player entity, this would be a non-used field for all non-player entities.
In my opinion, it would be better to add this field in the client class or GameStateClientData as you did